### DISCUSSION TITLE
Dynamic Paths

### DISCUSSION THREAD

**Discussion by enri90** [vikejs/vike#953] [Discussion]
it possible resolve paths dynamic '/*' ?

how can i use dynamic paths?
I run a query if the path url is present in the database I extract the page data

Es Dynamic urls:

@index or +Route.tsx
```typescript
export default '/*'
``` 
/
/about
/contact
/category/product
/article-name

Exist files +Page.ts
/login
/dashboard
/dashboard/users
/dashboard/products

**Comment by brillout** [comment]
I'm not sure what you mean.

**Comment by enri90** [comment]
the main pages of the site are generated by the page builder, maybe I'm wrong with the term dynamic path

**Reply by brillout** [reply]
Can you elaborate on your use case? I don't have enough information to be able to suggest a solution.
